### PR TITLE
Update-jwt-docs

### PR DESCRIPTION
**Pull request by donggyu04**
### Description

Updated JWT documents for [changes](https://github.com/opensearch-project/security/pull/4359).

> PR is in grogress

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _security/authentication-backends/jwt.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ Name | Description
 `jwt_url_parameter` | If the token is not transmitted in the HTTP header but rather as an URL parameter, define the name of the parameter here.
 `subject_key` | The key in the JSON payload that stores the username. If not set, the [subject](https://tools.ietf.org/html/rfc7519#section-4.1.2) registered claim is used.
 `roles_key` | The key in the JSON payload that stores the user's roles. The value of this key must be a comma-separated list of roles.
-`required_audience` | The name of the audience which the JWT must specify. This corresponds [`aud` claim of the JWT](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3).
+`required_audience` | The name of the audience which the JWT must specify. You can set single or multiple values by using the `,` delimeter such as `project1` or `project1,admin`. If you set multiple values, the JWT must have at least one required audience. This corresponds [`aud` claim of the JWT](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3).
```
Comment (kolchfa-aws) [inline _security/authentication-backends/jwt.md:125]:
```suggestion
`required_audience` | The name of the audience which the JWT must specify. You can set a single value (for example, `project1`) or multiple comma-separated values (for example, `project1,admin`). If you set multiple values, the JWT must have at least one required audience. This parameter corresponds to the [`aud` claim of the JWT](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3).
```

**Review by kolchfa-aws — APPROVED** [review]
Thank you, @donggyu04!

**Comment by kolchfa-aws** [comment]
@donggyu04 To what OpenSearch version(s) does this apply?

File: _security/authentication-backends/jwt.md [hunk 2]
```diff
@@ -122,7 +122,7 @@ Name | Description
 `jwt_url_parameter` | If the token is not transmitted in the HTTP header but rather as an URL parameter, define the name of the parameter here.
 `subject_key` | The key in the JSON payload that stores the username. If not set, the [subject](https://tools.ietf.org/html/rfc7519#section-4.1.2) registered claim is used.
 `roles_key` | The key in the JSON payload that stores the user's roles. The value of this key must be a comma-separated list of roles.
-`required_audience` | The name of the audience which the JWT must specify. This corresponds [`aud` claim of the JWT](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3).
+`required_audience` | The name of the audience which the JWT must specify. You can set a single value (for example, `project1`) or multiple comma-separated values (for example, `project1,admin`). If you set multiple values, the JWT must have at least one required audience. This parameter corresponds to the [`aud` claim of the JWT](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3).
```
Comment (natebower) [inline _security/authentication-backends/jwt.md:125]:
```suggestion
`required_audience` | The name of the audience that the JWT must specify. You can set a single value (for example, `project1`) or multiple comma-separated values (for example, `project1,admin`). If you set multiple values, the JWT must have at least one required audience. This parameter corresponds to the [`aud` claim of the JWT](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3).
```

**Review by natebower — COMMENTED** [review]
@donggyu04 @kolchfa-aws Just one minor change. Thanks!

**Comment by donggyu04** [comment]
> @donggyu04 To what OpenSearch version(s) does this apply?

I only know that it is version `2.x`, but I don't know the exact version or when it will be released.

> [PR](https://github.com/opensearch-project/security/pull/4359) has been merged :)

**Comment by kolchfa-aws** [comment]
@scrawfor99 Do you know what version this is for? Thanks!

**Pull request merged**